fieldmanager.php

```diff
@@ -74,6 +74,12 @@ function fieldmanager_load_class( $class ) {
 		return fieldmanager_load_file( 'util/class-fieldmanager-util-' . $class_id . '.php' );
 	}
 
+	if ( 0 === strpos( $class, 'Fieldmanager\Libraries' ) ) {
+		$classes  = explode( '\\', $class );
+		$class_id = strtolower( array_pop( $classes ) );
```

Comment:
while not critical for this PR, it would behoove us to be forward-thinking in this change and support n-depth namespaces/classes, so if the next library had the class `Fieldmanager\Libraries\Grand_Parent\Parent\Child` it would automatically be loaded at `libraries/grand-parent/parent/class-child.php`. We have lots of art for this, e.g. (parts of) the autoload generator in [mantle](https://github.com/alleyinteractive/mantle-framework/blob/main/src/autoload.php).

Reply:
Not a problem, updated.